config/routes.rb

```diff
@@ -5,6 +5,8 @@
     root :to => "api#index", :as => :entrypoint
     match "/", :to => "api#options", :via => :options
 
+    get "/ping" => "ping#index"
```

Comment:
/ping or /api/ping?

Reply:
or both?

Reply:
It's under the [api namespace](https://github.com/ManageIQ/manageiq-api/pull/39/files#diff-21497849d8f00507c9c8dcaf6288b136L4), so it's `/api/ping`